<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddItemToVault</class>
 <widget class="QWidget" name="AddItemToVault">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddItemToVault</string>
  </property>
  <widget class="QLabel" name="messagelabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
      <property name="text">
       <string>Item Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="deslineEdit">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
      <property name="text">
       <string>Weight</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="weilineEdit">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="itemnamelineEdit">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>-60</x>
     <y>-120</y>
     <width>811</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/guidesign/BackgroundImage.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Vault No</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="vaultnoEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>266</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>messagelabel</zorder>
  <zorder>pushButton</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>vaultnoEdit</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="finalbankingsystem.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AddItemToVault</receiver>
   <slot>addItemClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>-7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>AddItemToVault</receiver>
   <slot>backButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>-10</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addItemClicked()</slot>
  <slot>backButtonClicked()</slot>
 </slots>
</ui>
